packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -44,7 +45,8 @@ export class TriggerLauncher extends Component {
     this.state = {
       showTwoFAModal: false,
       trigger: initialTrigger,
-      error: triggersModel.getKonnectorJobError(initialTrigger)
+      error: triggersModel.getKonnectorJobError(initialTrigger),
+      running: get(initialTrigger, 'current_state.status') === 'running'
```

Comment:
Why do we duplicate the status here ? Why does TriggerLauncher does not let the downstream components get the trigger status ?

Reply:
getStatus should be a helper inside triggerModels. This could then be moved to cozy-client.

Reply:
> getStatus should be a helper inside triggerModels. This could then be moved to cozy-client.

Yep, good point, i'll make the change.

> Why does TriggerLauncher does not let the downstream components get the trigger status ?

Because the trigger status contained in `initialTrigger` doesn't change over time, even when the status trigger is started. TriggerLauncher is connected to cozy-realtime and makes sure downstream components always have an up-to-date status.

Reply:
> TriggerLauncher is connected to cozy-realtime and makes sure downstream components always have an up-to-date status.

TriggerLauncher could make sure downstream components always have an up-to-date trigger and job.